src/components/PaymentMethods/PaymentMethodsContainer/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { PaymentMethodsInput as PaymentInput } from '../PaymentMethodsInput/index'
+import { PaymentMethodsLabel as PaymentLabel } from '../PaymentMethodsLabel/index'
+import {
+  ContainerCard,
+  ContainerCardTitle,
+  PaymentContainer,
+  PaymentContainerItems,
+  PaymentContainerInstallment,
+  PaymentContainerInstallmentItems,
+  PaymentContainerInstallmentsLink,
+  LineDivisor,
+} from './style'
+import { PaymentMethodsProps } from './type'
+
+const PaymentMethods = ({
+  title,
+  children,
+  paymentData,
+}: PaymentMethodsProps) => {
```

Comment:
this part should not receive an array
it must be 1 only
and whoever uses it will make a map around it and assemble as many as they want

This makes it more expensive to use and loses flexibility

anything can call me